homeassistant/components/glances/coordinator.py

```diff
@@ -42,4 +44,31 @@ async def _async_update_data(self) -> dict[str, Any]:
             raise ConfigEntryAuthFailed from err
         except exceptions.GlancesApiError as err:
             raise UpdateFailed from err
+        # Update computed values
+        if data:
+            uptime = self.convert_uptime(data.get("uptime"))
+            data.update({"computed": {"uptime": uptime}})
```

Comment:
```suggestion
            data["uptime"] = self.convert_uptime(data.get("uptime"))
```
Just overwrite the value?

Reply:
Two reasons to have uptime in a nested dict : 
1- Having the computed value in a nested dict like the other sensors simplifies the code. 
With the nested dict, no need to make a special case for uptime anymore.
2- Keeping the original value (a relative  value stored as a `string`) and the computed value (an absolute `datetime`) distinct makes the code repeatable, no need to check for the type or risk of applying a transformation to the wrong value